plugins/text-freetype2/CMakeLists.txt

```diff
@@ -11,27 +11,52 @@ endif()
 
 find_package(Freetype REQUIRED)
 
+if(OS_MACOS)
+  find_package(Iconv REQUIRED)
+elseif(
+  OS_LINUX
+  OR OS_FREEBSD
+  OR OS_OPENBSD)
+  find_package(Fontconfig REQUIRED)
+endif()
+
 add_library(text-freetype2 MODULE)
 add_library(OBS::text-freetype2 ALIAS text-freetype2)
 
-target_sources(text-freetype2 PRIVATE find-font.h obs-convenience.c text-functionality.c text-freetype2.c
-                                      obs-convenience.h text-freetype2.h)
-target_link_libraries(text-freetype2 PRIVATE OBS::libobs Freetype::Freetype)
+target_sources(
+  text-freetype2
+  PRIVATE # cmake-format: sortable
+          $<$<PLATFORM_ID:Darwin>:find-font-cocoa.m>
+          $<$<PLATFORM_ID:Darwin>:find-font-iconv.c>
+          $<$<PLATFORM_ID:Darwin>:find-font.c>
+          $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:find-font-unix.c>
+          $<$<PLATFORM_ID:Windows>:find-font-windows.c>
+          $<$<PLATFORM_ID:Windows>:find-font.c>
```

Comment:
I might have asked before, but why:
```
$<$<PLATFORM_ID:Darwin>:find-font.c>
$<$<PLATFORM_ID:Windows>:find-font.c>
```
Instead of:
```
$<$<PLATFORM_ID:Darwin,Windows>:find-font.c>
```

Not a big deal, just curious.

Reply:
Just an oversight when the OS-specific branches were moved into generator expressions.